gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,5 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+org.gradle.daemon=false
```

Comment:
Do we really want to disable the daemon for all of us? Which problems did it cause for you?

Reply:
here it just causes some locking problems
but its fine for me to reenable it